monitoring/mongodb/alerts.yaml

```diff
@@ -97,20 +97,65 @@ groups:
   - alert: ReplicationLagWarning
     expr: |
       label_replace(
-        max(
-            max(mongodb_rs_members_optimeDate{namespace="${namespace}",pod=~"${service}.*",member_state="PRIMARY"})
-                by(pod, rs_nm)
-            - ignoring(member_idx) group_right
-            min(mongodb_rs_members_optimeDate{namespace="${namespace}",pod=~"${service}.*",member_state="SECONDARY"})
-                by(pod, rs_nm, member_idx)
-        ) by(member_idx, rs_nm) / 1000 > 30
-      , "member_idx", "$1", "member_idx", "(${service}[^.]*)\\.${service}.*")
-    for: 10m
+        max by(rs_nm, member_idx) (
+          (
+            max by(pod, rs_nm) (
+              mongodb_rs_members_optimeDate{namespace="${namespace}",pod=~"${service}.*",member_state="PRIMARY"}
+            ) - ignoring(member_idx) group_right
```

Comment:
`ignoring(member_idx)` is useless, since `max by(pod, rs_nm)` will keep only the `pod` and `rs_nm` labels